src/summary/patient-chart-level-two.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { RouteComponentProps } from "react-router";
+import { getCurrentPatient } from "@openmrs/esm-api";
+import styles from "./patient-chart-level-two.css";
+import PatientBanner from "./banner/patient-banner.component";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+import { AllergyCardLevelTwo } from "./history/allergy-card-level-two.component";
+
+export default function PatientChartLevelTwo(props: PatientChartLevelTwoProps) {
+  const [currentPatient, setCurrentPatient] = React.useState(null);
+  const [showPatientSummary, setShowPatientSummary] = React.useState(true);
+
+  React.useEffect(() => {
+    const subscription = getCurrentPatient().subscribe(patient => {
+      setCurrentPatient(patient);
+      createErrorHandler();
+    });
+
+    return () => subscription.unsubscribe();
+  });
+
+  return (
+    <main className="omrs-main-content">
+      <PatientBanner
+        match={props.match}
+        patient={currentPatient}
+        showPatientSummary={setShowPatientSummary}
+      ></PatientBanner>
+      {showPatientSummary && currentPatient && (
+        <div className={styles.patientSummary}>
+          <AllergyCardLevelTwo
```

Comment:
This should be wrapped in a Route so that it only shows when the url is `/patient/:patientUuid/chart/:allergy`. The PatientChartLevelTwo component seems to be generic to any of the level two components (not just allergy)

```tsx
<Route to={props.match + "/allergies"} render={p => (
  <AllergyCardLevelTwo currentPatient={currentPatient} />
)} />
```